packages/frontend/src/hooks/fungibleTokensIncludingNEAR.js

```diff
@@ -1,35 +1,26 @@
-import { useEffect, useState } from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector } from "react-redux";
 
-import { selectAccountId, selectBalance } from '../redux/slices/account';
-import { selectNearTokenFiatValueUSD } from '../redux/slices/tokenFiatValues';
-import { selectTokensWithMetadataForAccountId } from '../redux/slices/tokens';
+import { selectAccountId, selectBalance } from "../redux/slices/account";
+import { selectNearTokenFiatValueUSD } from "../redux/slices/tokenFiatValues";
+import { selectTokensWithMetadataForAccountId } from "../redux/slices/tokens";
 
-const fungibleTokensIncludingNEAR = (tokens, balance, nearTokenFiatValueUSD) => {
-    return [
-        {
-            balance,
-            symbol: 'NEAR',
-            usd: nearTokenFiatValueUSD
-        },
-        ...Object.values(tokens)
-    ];
+const useNEARAsTokenWithMetadata = () => {
+    const nearBalance = useSelector(selectBalance);
+    const nearTokenFiatValueUSD = useSelector(selectNearTokenFiatValueUSD);
+
+    return {
+        balance: nearBalance?.balanceAvailable || "",
+        onChainFTMetadata: { symbol: "NEAR" },
```

Comment:
Not a blocker, but for consistency, should there be an `icon` here, I wonder....